lib/msf/core/exploit/remote/winrm.rb

```diff
@@ -59,7 +59,7 @@ def winrm_run_cmd(cmd, timeout=20)
       return retval
     end
     shell_id = winrm_get_shell_id(resp)
-    resp = send_winrm_request(winrm_cmd_msg(cmd, shell_id),timeout)
+    resp = eend_winrm_request(winrm_cmd_msg(cmd, shell_id),timeout)
```

Comment:
I don't see where `eend_winrm_request` is defined. Is this a typo?

Reply:
Yep, typo - thanks.